torch/csrc/jit/api/function_impl.h

```diff
@@ -13,10 +13,14 @@ struct TORCH_API GraphFunction : public Function {
   GraphFunction(
       c10::QualifiedName name,
       std::shared_ptr<Graph> graph,
-      std::function<void(GraphFunction&)> function_creator)
+      std::function<void(GraphFunction&)> function_creator,
+      c10::optional<ExecutorExecutionMode> executor_execution_mode =
+          c10::nullopt)
       : name_(std::move(name)),
         graph_(std::move(graph)),
-        function_creator_(std::move(function_creator)) {}
+        function_creator_(std::move(function_creator)) {
+    executor_execution_mode_ = executor_execution_mode;
```

Comment:
why is this initialized inside a c-tor vs doing it like this: `graph_(std::move(graph)),`